ServerCore/Pages/Teams/Apply.cshtml.cs

```diff
@@ -37,6 +37,11 @@ public async Task<IActionResult> OnGet(int teamID, string password)
                 return Forbid();
             }
 
+            if (Event.IsInternEvent && TeamHelper.IsFTE(LoggedInUser.Email))
+            {
+                return NotFound("FTEs cannot sign up for this intern-only event.");
```

Comment:
This message should say something like "Microsoft employees and contractors"

Reply:
I realize that in this case, forbidden access made more sense.